Makefile

```diff
@@ -49,6 +49,16 @@ ruby: docker-image
 		--entrypoint bash ${PROTOC_IMAGE} \
 		-c "cd ./gen/pb-ruby && protoc -I/opt/include -I../../protos/ --ruby_out=lib ../../protos/*.proto"
 
+rust: docker-image
+	@echo "Generating rust protobuf files"
+	docker run \
+		--platform linux/amd64 \
```

Comment:
I'm not super familiar with Rust yet, but I assume this would only create a package for `linux/amd64`? Would we want to support other OS/architectures too?

Reply:
Oh, reading more it seems this is just the target OS/arch for the codegen, so when the docker image has run successfully, only Rust source code is being generated, no object files?

Reply:
> Oh, reading more it seems this is just the target OS/arch for the codegen, so when the docker image has run successfully, only Rust source code is being generated, no object files?

Correct! I think that `--platform` option actually shouldn't be needed at all; @jleightcap may have added it because of his local development environment.

(Rust package distributions are always source only, so the environment that creates them should have no effect on their buildability across other platforms.)

Reply:
Ah, this is a copy-paste from the other targets ([python](https://github.com/sigstore/protobuf-specs/blob/ae23636772323882df1dabd6d4cd42793ed211bb/Makefile#L30), [typescript](https://github.com/sigstore/protobuf-specs/blob/ae23636772323882df1dabd6d4cd42793ed211bb/Makefile#L39), &c).

The [`pb-rust` crate itself](https://github.com/sigstore/protobuf-specs/blob/ae23636772323882df1dabd6d4cd42793ed211bb/gen/pb-rust/Cargo.toml) is architecture agnostic.